ext/DiffTestsMKLSparseExt.jl

```diff
@@ -0,0 +1,25 @@
+module DiffTestsMKLSparseExt
+
+using LinearAlgebra: UpperTriangular, LowerTriangular
+using SparseArrays: sparse

```

Comment:
BTW this should not be done: Extensions should only import the package itself and the weak dependency but not even standard libraries.

Reply:
Why? SparseArrays and LinearAlgebra are the direct dependencies of DiffTests itself and MKLSparse. The latter provides more specific definitions of matrix multiplication and division for sparse matrices that are utilized by the test functions.

Reply:
You should import them indirectly via DiffTests and MKLSparse (see eg https://github.com/JuliaStats/LogExpFunctions.jl/pull/63).

Reply:
Thank you for the info. I've updated how the import is being done. However, the changes that the extensions makes to the function lists are not visible from the user session (I guess it's different world age of *DiffTests.jl* or something like that).